README - Getting started

patel-bhavin opened the pull request:
Adding a getting started section
Quick start commands
Adding explanation for our yaml files

Commit:
updating version 1

Commit:
updating recommendations

patel-bhavin committed:
Merge branch 'develop' into getting_started

patel-bhavin committed:
table format

patel-bhavin committed:
formatting

patel-bhavin committed:
updating commmands

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
updating readme for lookups

patel-bhavin committed:
testing

patel-bhavin committed:
updating format

patel-bhavin committed:
adding links

ljstella commented:
Couple of questions:

- Why are we putting them all in the readme.md as opposed to another file/folder and linking to it? 
- Do we think folks are going to continue to scroll that far to see them? 
- How will we ensure these stay up to date as the objects change?
- There are several other constraints on several of the fields in many files, that we've not indicated in these tables- are we planning to expand this more later, or just rely on validation failures to guide folks along?

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
adding yaml spec

patel-bhavin committed:
adding stories spec

patel-bhavin committed:
update readme

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
Branch was auto-updated.

patel-bhavin committed:
updating readme

patel-bhavin committed:
updating logo

patel-bhavin committed:
udpating formatting

patel-bhavin committed:
sb

patel-bhavin committed:
updating links

patel-bhavin committed:
udpating formatting

patel-bhavin commented:
Updated - 

- added the spec files to the folder and linked them
- rearranged the contents of the yamls
- when a developer makes yaml change, it will be expected of them to update this spec file and document that change. If it needs to be automated - we can consider that in the future. 
- also added enums but we also rely on contentctl to guide a user through the errors

patel-bhavin committed:
descirption

patel-bhavin committed:
note

patel-bhavin committed:
new image

patel-bhavin committed:
Merge branch 'develop' into getting_started

patel-bhavin commented:
Known failure in app inspect! merging this PR
